pkg/sentry/fsimpl/fuse/dev_test.go

```diff
@@ -15,31 +15,12 @@
 package fuse
 
 import (
-	"fmt"
-	"io"
-	"math/rand"
 	"testing"
 
-	"gvisor.dev/gvisor/pkg/abi/linux"
```

Comment:
See earlier comment. I don't think we need to make any of these testing changes.